src/ensembl/src/content/app/browser/browserHelper.ts

```diff
@@ -23,6 +31,13 @@ export function getChrLocationStr(
   return `${chrCode}:${startBp}-${endBp}`;
 }
 
+export const stringifyGenomeBrowserFocusId = (params: GenomeBrowserFocusIdConstituents) => 
+  buildEnsObjectId(params);
```

Comment:
Why do we need both?

Reply:
You mean why do we need both genome browser focus id and our internal id for use as a key when storing ensObject in redux?

The reason for this is to decouple genome browser ids from our internal ids. The format genome_id:type:stable_id should be enough for us to keep track of ensObjects in redux store; but we cannot predict whether genome browser's requirements for focus ids will change. Genome browser is essentially a different application from browser chrome; there is no point in coupling these two ids together. They are identical at the moment, but may not stay so in the future.